stigs/system/hsp-audit-stig-ubuntu-20-010164-newgrp.yaml

```
@@ -0,0 +1,17 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/ 
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: hsp-audit-stig-ubuntu-20-010164-newgrp
+spec:
+  severity: 5 
+  tags: ["STIG", "Ubuntu", "UBTU-20-010164", "newgrp"] 
+  message: "Alert! newgrp binary execution detected" 
+  process:
+    matchPaths:
+    - path: usr/bin/newgrp
```

```suggestion
  process:
    matchPaths:
    - path: /usr/bin/newgrp
```

I am not sure but does this use setgid? would monitoring setgid here make sense?

---

Yes, this uses uses setgid but will face the same problems as above